features/storage/nfs.feature

```diff
@@ -135,3 +139,5 @@ Feature: NFS Persistent Volume
     And the output should contain:
       | Permission denied |
 
+   Given I ensure "nfspd-<%= project.name %>" pod is deleted
+   And I ensure "nfsc-<%= project.name %>" pvc is deleted
```

Comment:
I don't mind these. But if test fails, we never reach these steps. Shouldn't we register deletion `after scenario` earlier as clean-up steps?

Reply:
@jhou1 ?

Reply:
Added that in https://github.com/openshift/verification-tests/pull/1562